dev-python/pyproject2setuppy/pyproject2setuppy-2.ebuild

```diff
@@ -4,7 +4,7 @@
 EAPI=7
 
 DISTUTILS_USE_SETUPTOOLS=manual
-PYTHON_COMPAT=( python{2_7,3_{5,6,7,8}} pypy{,3} )
+PYTHON_COMPAT=( python3_{5,6,7,8} pypy3 )
```

Comment:
Keep it here as long as setuptools supports py2. This was just added, and will be used by the eclass.

Reply:
Removed from PR